### DISCUSSION TITLE
Invoking the edit cast dialog directly through a URL

### DISCUSSION THREAD

**Discussion by qieman** [farcasterxyz/protocol#142] [Q&A] [answered]
Does Facaster have a feature to invoke the edit cast dialog directly through a URL, similar to Twitter's https://twitter.com/intent/tweet?text=create

**Comment by horsefacts** [comment, marked as answer]
We don't have cross-client compatible intents (yet!), but Warpcast provides cast intent URLs. Docs are [here](https://docs.farcaster.xyz/reference/warpcast/cast-composer-intents).
